src/timer/timer_enable.cc

```diff
@@ -313,6 +317,9 @@ void TimerReport::PrintToLog(unsigned indent_level) const {
     using std::chrono::duration_cast;
     using std::chrono::nanoseconds;
     const std::string indent(4 * indent_level, ' ');
+    if (!has_data_) {
+        return;
+    }
     LOG(INFO) << indent << "Section '" << GetSectionName() << "':";
     if (GetTotalHits() > 0) {
         LOG(INFO) << indent << "    hits    : " << GetTotalHits() << " times, freq: " << GetFreq() << " Hz";
```

Comment:
`XXX times at XXX Hz` would be shorter and more informative as it describe the relationship in-between.